### PR TITLE
draft for combining git and bagit

### PR DESCRIPTION
**Pull request by kba**

### CODE REVIEW

File: ocrd_gitzip.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+For space efficiency and to prevent accidental untracked changes, the git
+repository MUST be stored bare, i.e. without a manifested working tree, only
+containing the contents of the `.git` directory.
+
+## Invariants
+
+### Working tree must be OCRD-ZIP
+
+OCRD-GITZIP is strictly an extension of OCRD-ZIP, the working tree of the
+repository must satisfy all [invariants for OCRD-ZIP](ocrd_zip#invariants)
```
[the remark this answers is not in the document]
Comment (kba) [reply ocrd_gitzip.md]: Should be `#format`.

[the remark this answers is not in the document]
Comment (kba) [reply ocrd_gitzip.md]: No, I just forgot to push a change in the other branch where it's actually "Invariants".

**Comment by cneud** [comment]
@kba @VolkerHartmann Has the ``gitzip`` been replaced by BagIt now? Close or merge?

**Comment by kba** [comment]
We still lack a concept for versioning, that's what this PR originally set out to solve. It's more an extension to the BagIt approach that adds a temporary dimension.

That being said, unless @VolkerHartmann wants to go in that direction with versioning, it's probably a dead proposal and PR should be closed.

**Comment by cneud** [comment]
Thx for elaborating @kba, will wait for @VolkerHartmann then.

**Comment by kba** [comment]
This is not going anywhere, closing.

**Pull request closed without merging**